omx/gstomxaacenc.c

```diff
@@ -312,6 +312,25 @@ gst_omx_aac_enc_set_format (GstOMXAudioEnc * enc, GstOMXPort * port,
     return FALSE;
   }
 
+  GST_OMX_INIT_STRUCT (&aac_profile);
+  aac_profile.nPortIndex = enc->enc_out_port->index;
+
+  err =
+      gst_omx_component_get_parameter (enc->enc, OMX_IndexParamAudioAac,
+      &aac_profile);
+  if (err != OMX_ErrorNone) {
+    GST_ERROR_OBJECT (self,
+        "Failed to get AAC parameters from component: %s (0x%08x)",
+        gst_omx_error_to_string (err), err);
+    return FALSE;
+  } else {
+    GST_DEBUG_OBJECT (self,
+        "Audio AAC parameters: eAACProfile=%d (OMX_AUDIO_AACObjectLC=%d), eAACStreamFormat=%d (OMX_AUDIO_AACStreamFormatMP4ADTS=%d), nSampleRate=%d, nChannels=%d, nBitRate=%d",
+        (int) aac_profile.eAACProfile, (int) OMX_AUDIO_AACObjectLC,
```

Comment:
use Glib types instead. They are portable across platforms. This is `g_int` instead  of `int`.